### PR TITLE
Add back null receiver to alertmanager example config

### PR DESCRIPTION
**Pull request by MPV**
See problem description in https://github.com/mr-karan/calert/pull/3

TLDR:
Since Helm merges the default alertmanager config from the `prometheus-operator`(which [has a null receiver on the root route](https://github.com/helm/charts/blob/1e6a30c/stable/prometheus-operator/values.yaml#L131)), you get this error when using your [currently suggested alertmanager config](https://github.com/patrickmslatteryvt/calert/tree/master/calert#alertmanager-configuration):

`level=error ts=2020-04-21T09:15:12.828Z caller=coordinator.go:124 component=configuration msg="Loading configuration file failed" file=/etc/alertmanager/config/alertmanager.yaml err="undefined receiver \"null\" used in route"`

This PR makes sure to keep including that `"null"` receiver, so it doesn't break like that.

### CODE REVIEW

**Comment by MPV** [comment]
Alternatively, we could suggest the `google-chat` receiver as the receiver for the top-level route:

```diff
alertmanager:
  config:
    route:
-     receiver: "null"
+     receiver: "google-chat"
      routes:
      - match:
          severity: critical
        receiver: google-chat
        group_by: [alertname]
    receivers:
-   - name: "null"
    - name: 'google-chat'
      webhook_configs:
      - url: "http://calert.clu-inf-all.svc.cluster.local:6000/create?room_name=<room>"
```

However, if I understand it right, that would mean that any (not just `critical`) would be sent to calert?

Thus, maybe what I've done in this PR is the best suggestion (for now)?

**Comment by MPV** [comment]
Closing as https://github.com/mr-karan/calert/pull/14 was merged instead.
